core/controllers/admin_test.py

```diff
@@ -2451,18 +2577,21 @@ def setUp(self):
         self.admin_user_id = self.get_user_id_from_email(self.ADMIN_EMAIL)
 
     def test_delete_without_user_id_raises_error(self):
+        """test delete without user id, raises error."""
         self.delete_json(
             '/deleteuserhandler',
             params={'username': 'someusername'},
             expected_status_int=400)
 
     def test_delete_without_username_raises_error(self):
+        """test delete without username, raises error."""
```

Comment:
I don't find value in these comments as it just repeats the test name. Do we really need them (IIRC, we never checked for docstrings in the test file)